content/number-theory/chapter.tex

```diff
@@ -47,10 +47,11 @@ \section{Primes}
 	Primitive roots exist modulo any prime power $p^a$, except for $p = 2, a > 2$, and there are $\phi(\phi(p^a))$ many.
 	For $p = 2, a > 2$, the group $\mathbb Z_{2^a}^\times$ is instead isomorphic to $\mathbb Z_2 \times \mathbb Z_{2^{a-2}}$.
 
-\section{Estimates}
-	$\sum_{d|n} d = O(n \log \log n)$.
+\section{Highly composite numbers}
+The number of divisors of $n$ is $O(\log(\log(n)))$. Max number of divisors up to $10^n$ (n, number of divisors, number):
+\import{hcn.tex}
```

Comment:
Está tabla no me convence mucho, no se podrá hacer como la de los facotoriales, poniendo el n, number of divisors, number a la izquierda, pero poniendolo solo una ves?